lib/KBox/src/KMessage.h

```diff
@@ -191,6 +196,94 @@ class IMUMessage: public KMessage {
     };
 };
 
+class NAVMessage: public KMessage {
+  private:
+    bool apMode, apHeadingMode, apWaypointMode, apDodgeMode;
+    double currentHeading, targetHeading, courseToWaypoint;
```

Comment:
Yes, its common on Autopilots.  A seadoo is on a collision course and though you are tempted to hit it, you decide not to, go into dodge mode, manually steer for a few seconds to avoid, then go back out of dodge mode and resume the previous heading.  Or a series of waves pushes you off course and you decide to give the AP a break, go into dodge mode, steer back onto the proper heading, then go back out of dodge mode to resume normal AP operation.  Before going into Dodge mode we could have been in Heading Mode or Waypoint mode.  In Dodge mode, we stop the AP controlling the rudder, although the device continues to calculate where the rudder should be.